src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -56,6 +59,20 @@
     public static final String MESSAGE_SUCCESS_BOOKING = "New booking added: %1$s";
     public static final String MESSAGE_DUPLICATE_BOOKING = "This booking already exists in the address book";
 
+    public static final String COMMAND_WORD_INGREDIENT = "addingredient";
+    public static final String COMMAND_ALIAS_INGREDIENT = "ia";
```

Comment:
Should probably standardise; I have used "ap" and "ab" for addbooking and addperson.
Personally I find "ai" more intuitive than "ia" since it follows the order of the words.

Also, I think calling the command "addIngredient" might be confusing- does add mean replenish stock or creating a new ingredient?

I understand that you intend to create a new ingredient here, but this terminology might be potentially confusing for end users. Perhaps you might want to call it "newIngredient" as displayed to the user....?

We need to have a discussion on this.

Reply:
The original intention for using 'ia' for addIngredient is for the 'i' to act as a 'prefix' for ingredient.
Basically, we have four major components - member, booking, inventory, staff - so we can use 'm', 'b', 'i', 's' for the aliases for each of the four components respectively. But yes, I see your point on how 'ai' is more intuitive.